type-crate/src/types/group.rs

```diff
@@ -44,12 +44,18 @@ pub struct MarginfiGroup {
     /// Can modify the emissions `flags`, `emissions_rate` and `emissions_mint`, but nothing else,
     /// for every bank under this group
     pub delegate_emissions_admin: Pubkey,
+    /// Can run bankruptcy and forced deleverage ixes to e.g. sunset risky/illiquid assets
+    pub risk_admin: Pubkey,
```

Comment:
Just noticed this comes before the panic state cache so it would corrupt that data in prod, but we don't currently use panic state cache in prod, so it's actually fine. Nonetheless, gna move it after the cache so it doesn't step on the prod data by chance.